src/components/ebay-breadcrumb/mock/index.js

```diff
@@ -0,0 +1,42 @@
+const item = { href: 'https://www.ebay.com/', navSrc: '{"actionKind":"NAVSRC","operationId":"2489527"}', _sp: 'p2489527.m4340.l9751.c1' };
+
+const getItem = (text, href) => {
+    const clonedItem = JSON.parse(JSON.stringify(item));
+    clonedItem.renderBody = (stream) => stream.write(text);
+    clonedItem.href = href;
+    return clonedItem;
+};
+const items = [getItem('eBay', 'https://www.ebay.com'),
```

Comment:
You may want to call this something like `basicItems` instead of `items`, since `items` is already used for the nested `items` array.